sections/memory_layout.tex

```diff
@@ -1,39 +1,51 @@
 \section{Block Packed Storage}\label{sec: memory layout}
+Single-step updates are useful in small scale settings to understand the local behaviour of the random field or if numerical stability is an issue. For the analysis of a large number of evaluations, however, this isn't the most efficient solution as we can't rely on BLAS-3 operations in the update process. To avoid this drawback our implementation offers bulk updates, which admit a performance boost while also preserving theoretical accuracy.
```

Comment:
Ist ein single-step update wirklich numerisch genauer? Es passieren doch die gleichen rechenoperationen nur in einer anderen reihenfolge.

Reply:
Sollte doch genauer sein, weil du nicht die Wurzel aus dem ganzen Block berechnen musst? Würde annehmen, dass sich in einer Cholesky-Zerlegung die numerischen Fehler akkumulieren

Reply:
wenn deine block-size 4x4 sind, musst du ja entweder 4 reihen nacheinander machen (4 wurzeln, wo du immer die reihe davor verwendest für die nächste), oder du machst eine cholesky zerlegung von dem eckblock (die dann effektiv auch diese 4 wurzeln zieht. Wir ändern wirklich nur die Reihenfolger der operationen. Außer man verwendet so was wie pivoted cholesky oder so und nicht die standard cholesky